ladybug_comfort/map/mrt.py

```diff
@@ -12,6 +13,7 @@
 from ladybug.analysisperiod import AnalysisPeriod
 from ladybug.header import Header
 from ladybug.datacollection import HourlyContinuousCollection
+from honeybee_radiance_postprocess.reader import binary_to_array
```

Comment:
Yea, I think we should just copy this function somewhere either into this mrt.py module or into some other _helper.py module in this map sub-package if you think that it might be used by the other modules here.

Otherwise, we would need to include honeybee-radiance-postprocess in the ladybug-comfort Docker image for this to run successfully with queenbee-argo.